src/pages/RestaurantsPage/RestaurantsPage.tsx

```diff
@@ -0,0 +1,192 @@
+import { useState, useRef } from "react";
+import Dropdown from "../../components/Dropdown/Dropdown";
+import RestaurantCard from "../../components/RestaurantCard/RestaurantCard";
+import { SectionTitle } from "../../components/SectionTitle/SectionTitle";
+import { useFetch } from "../../hooks/useFetch";
+import useIsMobile from "../../hooks/useIsMobile";
+import useIsTablet from "../../hooks/useIsTablet";
+import { Restaurant } from "../../types/types";
+import styles from "./RestaurantsPage.module.scss";
+
+const RestaurantsPage = () => {
+  const data = useFetch();
+  const restaurantsData: Restaurant[] = data.restaurants;
+  const isTablet = useIsTablet();
+  const isMobile = useIsMobile();
+  const isMobileOrTable = isMobile || isTablet;
+  const buttonRefs: React.MutableRefObject<HTMLButtonElement | null>[] = [
+    useRef(null),
+    useRef(null),
+    useRef(null),
+    useRef(null),
+  ];
```

Comment:
I highly doubt you really need to use `refs` in here.

Reply:
@nitzanpap done